test/runtests.jl

```diff
@@ -19,30 +19,59 @@ img_eps = isinteractive() ? 1e-2 : 10e-2
 #     image_comparison_facts(:gadfly, skip=[4,6,23,24,27], eps=img_eps)
 # end
 
+facts("GR") do
+    @fact gr() --> Plots.GRBackend()
+    @fact backend() --> Plots.GRBackend()
+
+    image_comparison_facts(:gr, eps=img_eps)
+end
+
 facts("PyPlot") do
     @fact pyplot() --> Plots.PyPlotBackend()
     @fact backend() --> Plots.PyPlotBackend()
 
-    image_comparison_facts(:pyplot, skip=[6,25,30,31], eps=img_eps)
+    image_comparison_facts(:pyplot, skip=[2,31], eps=img_eps)
 end
 
-facts("GR") do
-    @fact gr() --> Plots.GRBackend()
-    @fact backend() --> Plots.GRBackend()
-
-    if is_linux() && isinteractive()
-        image_comparison_facts(:gr, skip=[2,25,30,31], eps=img_eps)
+facts("PlotlyJS") do
+    @fact plotlyjs() --> Plots.PlotlyJSBackend()
+    @fact backend() --> Plots.PlotlyJSBackend()
+
+    if is_linux()
+        image_comparison_facts(:plotlyjs,
+            skip=[
+                2,  # animation
+                10, # match_dimensions not defined
```

Comment:
I think that this testimage really errored. I have look at the exact error message when I'm at home. Otherwise, I'm not sure whether we don't want warnings in the tests. They still pass.